tests/attacks/inference/test_membership_inference.py

```diff
@@ -0,0 +1,363 @@
+# MIT License
+#
+# Copyright (C) The Adversarial Robustness Toolbox (ART) Authors 2020
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit
+# persons to whom the Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the
+# Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+# WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+# TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import logging
+import pytest
+
+import numpy as np
+import torch.nn as nn
+import torch.optim as optim
+
+import keras
+
+from art.attacks.inference import (
+    MembershipInferenceBlackBoxRuleBased,
+    MembershipInferenceBlackBox,
+)
+from art.estimators.classification.pytorch import PyTorchClassifier
+from art.estimators.classification.keras import KerasClassifier
+from art.estimators.estimator import BaseEstimator
+
+from tests.attacks.utils import backend_test_classifier_type_check_fail
+
+
+logger = logging.getLogger(__name__)
+attack_train_ratio = 0.5
+num_classes_iris = 3
+num_classes_mnist = 10
+
+
+@pytest.fixture()
+def fix_get_mnist_subset(get_mnist_dataset):
```

Comment:
this fixture needs to be removed and instead the project-wide `get_default_mnist_subset` should be used